src/system/SystemPacketBuffer.cpp

```diff
@@ -565,7 +583,7 @@ PacketBufferHandle PacketBufferHandle::New(size_t aAvailableSize, uint16_t aRese
     UNLOCK_BUF_POOL();
 
 #elif CHIP_SYSTEM_PACKETBUFFER_FROM_CHIP_HEAP
-
+    const size_t lBlockSize = PacketBuffer::kStructureSize + lAllocSize;
```

Comment:
So this is safe because that addition on the right is just `sumOfSizes` which we already checked fits in size_t.

But it might be clearer to just do `static_cast<size_t>(sumOfSizes)` with a comment about how that's kStructureSize + lAllocSize.

Reply:
Yes, they are the same. Will change it to sumOfSizes.